go.sum

```diff
@@ -90,6 +90,10 @@ github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d/go.mod h1:rBZYJk5
 github.com/antihax/optional v1.0.0/go.mod h1:uupD/76wgC+ih3iEmQUL+0Ugr19nfwCT1kdvxnR2qWY=
 github.com/antlr/antlr4/runtime/Go/antlr v0.0.0-20210826220005-b48c857c3a0e h1:GCzyKMDDjSGnlpl3clrdAK7I1AaVoaiKDOYkUzChZzg=
 github.com/antlr/antlr4/runtime/Go/antlr v0.0.0-20210826220005-b48c857c3a0e/go.mod h1:F7bn7fEU90QkQ3tnmaTx3LTKLEDqnwWODIYppRQ5hnY=
+github.com/aojea/rwconn v0.0.0-20220622092013-13e3141a2a4e h1:rldE7HndxqaNyukzKCUFnWEgwt5WtwESLt6evEvoeHw=
+github.com/aojea/rwconn v0.0.0-20220622092013-13e3141a2a4e/go.mod h1:LUO0QX1YNsA51BR48slR87GsvEMiTTOWNdC6aoG+BTA=
```

Comment:
Old version here, I think you need to `go mod tidy` to clean this up?

Reply:
oh, good catch